src/hooks/useVoicePlayer/index.tsx

```diff
@@ -56,13 +56,13 @@ export const VoicePlayerProvider = (props: VoicePlayerProps): React.ReactElement
     };
     audioPlayer.onpause = () => {
       eventHandler?.onPlayingStopped({
-        playbackTime: audioPlayer.currentTime * 1000,
-        playSize: audioPlayer.duration * 1000,
+        playbackTime: audioPlayer.currentTime,
+        playSize: audioPlayer.duration,
         audioFile: audioFile,
       });
     };
     audioPlayer.ontimeupdate = () => {
-      eventHandler?.onPlaybackTimeUpdated(audioPlayer.currentTime * 1000);
```

Comment:
```javascript
* 1000
```
This operation causes `ontimeupdated` event of the Audio(Player)
The weird behavior is that the timestamp is updated to the duration of the audio, so it finished the audio play when playing the audio from the paused state.

Let me leave this comment to study later